src/components/EditPost.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from "react";
+import { db } from "../firebase";
+import "./EditPost.css";
+import { updateDoc, doc } from "firebase/firestore";
+
+const EditPost = ({ editingId, onEditDone, onEditSubmit }) => {
+  const [editTitle, setEditTitle] = useState();
+  const [editText, setEditText] = useState();
+  const [docRef, setDocRef] = useState(null);
+
+  useEffect(() => {
+    if (editingId) {
+      setDocRef(doc(db, "posts", editingId));
+    }
+  }, [editingId]);
+
+  // console.log(docRef);
+
+  const editDocument = async () => {
+    if (docRef) {
+      await updateDoc(docRef, {
+        title: editTitle,
+        postsText: editText,
+      });
+    }
+
+    onEditSubmit({ id: editingId, title: editTitle, postsText: editText });
+    onEditDone();
+  };
+
+  if (!editingId) return null;
```

Comment:
結構好みの問題ではあるのですが、if文の中身が複数行の時と、１行の時でコードの見た目が変わるのは一貫性がなくなってしまうので以下のような変更はいかがでしょうか？
```suggestion
  if (!editingId) {
    return null;
  }
```

Reply:
こちら修正しました！